doc/nrf/ug_nrf5340.rst

```diff
@@ -162,6 +150,13 @@ The sample implements the RPMsg transport using the `OpenAMP`_ library to commun
 
 When working with 802.15.4-based protocols like Thread or Zigbee, program the :ref:`zephyr:nrf-ieee802154-rpmsg-sample` sample to the network core to run :ref:`Thread and Zigbee samples <samples>` on nRF5340.
 
+Configuration of network samples
+--------------------------------
+
+When a network sample is built automatically as a child image in a multi-image build (see :ref:`ug_nrf5340_building`), you can define the relevant Kconfig options (if required) in :file:`<application_core_sample_location>/child_image/<network_sample>.conf`.
```

Comment:
We should not use <...> variables. How about:
```suggestion
When a network sample is built automatically as a child image in a multi-image build (see :ref:`ug_nrf5340_building`), you can define the relevant Kconfig options (if required) in a :file:`.conf` file. 
Name the file *network_sample*\ .conf, where *network_sample* is the name of the child image (for example, ``hci_rpmsg.conf``).
Place the file in a :file:`child_image` subfolder of the application sample directory.
```

Reply:
Updated.